tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -52,17 +52,24 @@ def do_build(args: CensusBuildArgs, skip_completed_steps: bool = False) -> int:
     logging.info(f"Census build: start [version={__version__}]")
     logging.info(args)
 
-    build_steps: List[Callable[[CensusBuildArgs], bool]] = [
-        do_prebuild_set_defaults,
-        do_prebuild_checks,
-        do_build_soma,
-        do_validate_soma,
-        do_create_reports,
-        do_data_copy,
-        do_the_release,
-        do_report_copy,
-        do_old_release_cleanup,
-    ]
+    # Infer steps from environment variables, if provided
+    if os.getenv("CENSUS_BUILD_STEPS"):
+        steps = os.getenv("CENSUS_BUILD_STEPS", "").split(",")
+        build_steps = [globals()[step] for step in steps]
+        logging.info(f"Census build: using steps {steps}")
+    else:
+        build_steps = [
+            do_prebuild_set_defaults,
+            do_prebuild_checks,
+            do_build_soma,
+            do_validate_soma,
+            do_create_reports,
+            do_data_copy,
+            do_the_release,
+            do_report_copy,
+            do_old_release_cleanup,
+            do_log_copy,
```

Comment:
This needs to move here, since it is now part of the build proper. Sub-steps will not require to copy logs.